rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/evaluation/MuteExecutionTest.kt

```diff
@@ -75,638 +70,26 @@ class MuteExecutionTest {
-        assertEquals(listOf("Hello World!"), si.displayed)
-        assertEquals(logHandler.getExecutedSubroutines().size, 0)
-    }
-
-    @Test
-    fun executeCallToFibonacciWrittenInRpg() {
-        val cu = assertASTCanBeProduced("CALCFIBCAL", true)
-        cu.resolve()
-        val si = CollectorSystemInterface()
-        val logHandler = ListLogHandler()
-        si.programs["CALCFIB"] = rpgProgram("CALCFIB")
-        execute(cu, mapOf("ppdat" to StringValue("10")), si, listOf(logHandler))
-        assertEquals(listOf("FIBONACCI OF: 10 IS: 55"), si.displayed)
-        assertEquals(1, logHandler.getExecutedSubroutines().size)
-    }
-
-    @Test
-    fun executeCallToFibonacciWrittenOnTheJvm() {
-        val cu = assertASTCanBeProduced("CALCFIBCAL", true)
-        cu.resolve()
-        val si = CollectorSystemInterface()
-        val logHandler = ListLogHandler()
-        si.programs["CALCFIB"] = object : JvmProgramRaw("CALCFIB", listOf(ProgramParam("ppdat", StringType(8)))) {
-            override fun execute(systemInterface: SystemInterface, params: LinkedHashMap<String, Value>): List<Value> {
-                val n = params["ppdat"]!!.asString().valueWithoutPadding.asInt()
-                var t1 = 0
-                var t2 = 1
-
-                for (i in 1..n) {
-                    val sum = t1 + t2
-                    t1 = t2
-                    t2 = sum
-                }
-                systemInterface.display("FIBONACCI OF: $n IS: $t1")
-                return listOf(params["ppdat"]!!)
-            }
-        }
-        execute(cu, mapOf("ppdat" to StringValue("10")), si, listOf(logHandler))
-        assertEquals(listOf("FIBONACCI OF: 10 IS: 55"), si.displayed)
-        assertEquals(logHandler.getExecutedSubroutines().size, 0)
+        assertEquals(1, cu.main.stmts[0].muteAnnotations.size)
+        val interpreter = execute(cu, emptyMap())
+        assertEquals(1, interpreter.systemInterface.getExecutedAnnotation().size)
+        val muteAnnotationExecuted = interpreter.systemInterface.getExecutedAnnotation().values.first()
+        assertFalse(muteAnnotationExecuted.succeeded())
+        assertEquals("This code should not be executed", muteAnnotationExecuted.value1Result.render())
     }
 
     @Test
-    fun executeFibonacciWrittenInRpgAsProgram() {
-        val cu = assertASTCanBeProduced("CALCFIB", true)
+    fun parsingSIMPLE_MUTE_FAIL_EVALUATED_MESSAGE() {
+        val cu = assertASTCanBeProduced("mute/SIMPLE_MUTE_FAIL_EVALUATED_MESSAGE", true, withMuteSupport = true)
         cu.resolve()
-        val si = CollectorSystemInterface()
-        val rpgProgram = RpgProgram(cu)
-        rpgProgram.execute(si, linkedMapOf("ppdat" to StringValue("10")))
-        assertEquals(1, rpgProgram.params().size)
-        assertEquals(ProgramParam("ppdat", StringType(8)), rpgProgram.params()[0])
-        assertEquals(listOf("FIBONACCI OF: 10 IS: 55"), si.displayed)
-    }
-
-    @Test
-    fun executeHELLOCASE() {
```

Comment:
Why are we removing all these tests?

Reply:
They were the same test we have in InterpreterTest class